### PR TITLE
Added initial acceptance test for application submission

### PR DESCRIPTION
**Pull request by stuartmccoll**
This commit contains an initial acceptance test for the successful submission of an application.

I have abstracted methods for each part of the project application form, which will allow us to easily test other paths through this part of the service without re-writing code.

### CODE REVIEW

**Comment by paulmsmith** [comment]
Running `RSpec` tests reported no failures.
Running `rails test` reported some issues. Hopefully quick to fix.

**Comment by stuartmccoll** [comment]
> Running `RSpec` tests reported no failures.
> Running `rails test` reported some issues. Hopefully quick to fix.

Ready for you to take a look at again, @paulmsmith. I've removed all of the Minitest tests and replaced them with RSpec equivalents. No need to run `rails test` anymore, just `rspec`. I've removed the running of `rails test` from the GitHub Action too.

**Comment by paulmsmith** [comment]
```
 rspec
...........................................................

Finished in 5.75 seconds (files took 2.42 seconds to load)
59 examples, 0 failures
```
Ran `rspec` locally and no errors.

File: spec/controllers/released_form_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  describe "POST #receive" do
+
+    it "should release the permission to start from the webhook" do
+
+      project = create(
+          :project,
+          id: '2c660111-ab15-4221-98e0-cf0e02748a9b'
+      )
+
+      post :receive, params: {
```
Comment (paulmsmith) [inline spec/controllers/released_form_controller_spec.rb]: Assuming this is some kind of mock data for the test? :)

**Pull request merged**